packages/grid/_modules_/grid/models/api/localeTextApi.ts

```diff
@@ -2,12 +2,65 @@
  * Set the types of the texts in the grid.
  */
 export interface LocaleText {
+  // Root
+  rootGridLabel: string;
+
   // Density selector toolbar button text
   toolbarDensity: React.ReactNode;
   toolbarDensityLabel: string;
   toolbarDensityCompact: string;
   toolbarDensityStandard: string;
   toolbarDensityComfortable: string;
+
+  // Columns selector toolbar button text
+  toolbarColumns: React.ReactNode;
+  toolbarColumnsLabel: string;
+
+  // Filters toolbar button text
+  toolbarFilters: React.ReactNode;
+  toolbarFiltersLabel: string;
+  toolbarFiltersTooltipHide: React.ReactNode;
+  toolbarFiltersTooltipShow: React.ReactNode;
+  toolbarFiltersTooltipActive: Function;
+
+  // Columns panel text
+  columnsPanelTextFieldLabel: string;
+  columnsPanelTextFieldPlaceholder: string;
+  columnsPanelDragIconLabel: string;
+  columnsPanelShowAllButton: React.ReactNode;
+  columnsPanelHideAllButton: React.ReactNode;
+
+  // Filter panel text
+  filterPanelAddFilter: React.ReactNode;
+  filterPanelDeleteIconLabel: string;
+  filterPanelOperators: React.ReactNode;
+  filterPanelOperatorAnd: React.ReactNode;
+  filterPanelOperatorOr: React.ReactNode;
+  filterPanelColumns: React.ReactNode;
+
+  // Column menu text
+  columnMenuLabel: string;
+  columnMenuShowColumns: React.ReactNode;
+  columnMenuFilter: React.ReactNode;
+  columnMenuHideColumn: React.ReactNode;
+  columnMenuUnsort: React.ReactNode;
+  columnMenuSortAsc: React.ReactNode;
+  columnMenuSortDesc: React.ReactNode;
+
+  // Column header text
+  columnHeaderFiltersTooltipActive: Function;
+  columnHeaderFiltersLabel: string;
+  columnHeaderSortIconLabel: string;
+
+  // Rows selected footer text
+  footerRowSelected: React.ReactNode;
```

Comment:
No longer used? 

```suggestion
```

Reply:
Yup, but I removed the other one and keep this one as the name is more appropriate.